python/src/functions/generate_treasury_report.py

```diff
@@ -267,6 +282,28 @@ def process_event(
     return {"statusCode": 200, "body": "Success"}
 
 
+def delete_output_file(
+    s3_client: S3Client,
+    organization: OrganizationObj,
+    project_use_code: str,
+    logger: structlog.stdlib.BoundLogger,
+):
+    try:
+        delete_file_from_s3(
```

Comment:
Note: we also need to delete the `report.zip` file in the same directory in addition to this file

Reply:
The file is created here: https://github.com/usdigitalresponse/cpf-reporter/blob/main/python/src/functions/create_archive.py#L104

Reply:
This should be the list of files to delete:
```
        delete_file_from_s3(
            client=s3_client,
            bucket=os.environ["REPORTING_DATA_BUCKET_NAME"],
            key=get_generated_output_file_key(
                file_type=OutputFileType.XLSX,
                project=project_use_code,
                organization=organization,
            ),
        )
        delete_file_from_s3(
            client=s3_client,
            bucket=os.environ["REPORTING_DATA_BUCKET_NAME"],
            key=get_generated_output_file_key(
                file_type=OutputFileType.CSV,
                project=project_use_code,
                organization=organization,
            ),
        )
        delete_file_from_s3(
            client=s3_client,
            bucket=os.environ["REPORTING_DATA_BUCKET_NAME"],
            key=get_generated_output_file_key(
                file_type=OutputFileType.JSON,
                project=project_use_code,
                organization=organization,
            ),
        )
        report_key = f"treasuryreports/{organization.id}/{organization.preferences.current_reporting_period_id}/report.zip"
        delete_file_from_s3(
            client=s3_client,
            bucket=os.environ["REPORTING_DATA_BUCKET_NAME"],
            key=report_key,
        )
```

Reply:
Added, but I'm looking into fixing the issue with the CI; (all the changed files have a `python/` prepended that needs to be removed)

Reply:
Should be fixed here: https://github.com/usdigitalresponse/cpf-reporter/pull/575 🤞🏼